project/AppDependencies.scala

```diff
@@ -3,13 +3,13 @@ import sbt._
 object AppDependencies {
   import play.core.PlayVersion
 
+  val mongoplay_ver = "0.73.0"
```

Comment:
One small comment. Other than that this looks good to me.

Can you just make this camelCase for consistency. just something like
`val mongoVersion = "0.73.0"`